doc/source/whatsnew/v0.18.1.txt

```diff
@@ -105,6 +104,50 @@ API changes
 - ``pd.show_versions()`` now includes ``pandas_datareader`` version (:issue:`12740`)
 - Provide a proper ``__name__`` and ``__qualname__`` attributes for generic functions (:issue:`12021`)
 
+.. _whatsnew_0181.numpy_compatibility
+
+Compatibility between pandas array-like methods (e.g. ```sum`` and ``take``) and their ``numpy``
+counterparts has been greatly increased by augmenting the signatures of the ``pandas`` methods so
+as to accept arguments that can be passed in from ``numpy``, even if they are not necessarily
+used in the ``pandas`` implementation (:issue:`12644`). An example of this augmentation is
+illustrated below:
```

Comment:
Do you just mean move the section where I listed the issues?  Or do you also want an enumeration of all of the compatibility problems that I squashed?